addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -0,0 +1,78 @@
+# Onkyo Binding
+
+This binding integrates the Onkyo AV receivers.
+
+## Introduction
+Binding should be compatible with Onkyo AV receivers which support ISCP (Integra Serial Control Protocol) over Ethernet (eISCP).
+
+## Supported Things
+This binding supports only one thing: The Onkyo AV Receiver
+
+
+## Discovery
+This binding can discover the supported Onkyo AV Receivers. At the moment only the following models are supported:
+
+* TX-NR535
```

Comment:
I just tested it on my TX-NR616 and it works. Due to your supported model check it looks like it's a code change [here](https://github.com/openhab/openhab2-addons/pull/768/files#diff-6a4118c696d2ddc35356333392a8f11bR27) every time a model is verified as working.